### PR TITLE
[R20-948] Grants listing

### PR DESCRIPTION
**Pull request by jeffdowdle**
**Issue**: R20-948

### What I did

- Fixed various issues in how grants displayed in search results
- Consolidated common utill functions
- Refactored the display of the grant meta display to share code between the grant pages and grant search results.

### How to test

- Cypress
- Create a grants listing page in drupal

#### For all PR's

- [ ] I've added relevant changes to the project Readme if needed.
- [ ] I've updated the documentation site as needed.
- [ ] I have added unit tests to cover my changes (if not applicable, please state why in a comment)

#### For new components only

- [ ] I have added a story covering all variants
- [ ] I have checked a11y tab in storybook passes
- [ ] Any events are emitted on the event bus

### CODE REVIEW

File: packages/nuxt-ripple/package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
   "exports": {
     ".": "./nuxt.config.ts",
     "./mapping": "./mapping/index.ts",
-    "./robots": "./robots.config.ts"
+    "./robots": "./robots.config.ts",
+    "./utils": "./utilities/index.ts"
```
Comment (dylankelly) [inline packages/nuxt-ripple/package.json:12]: can we rename this - I understand why they can be exported through `/utils` as they will be auto imported, but the similarity in names is confusing. Could we call it `/lib` instead?

**Review by dylankelly — CHANGES_REQUESTED** [review]
@jeffdowdle looks like the re order of the grants items needs to be changed in the grants page test

File: packages/nuxt-ripple/package.json (hunk 1 above)
Comment (jeffdowdle) [reply packages/nuxt-ripple/package.json:12]: @dylankelly  like this? `"./lib": "./utilities/index.ts"`

Comment (dylankelly) [reply packages/nuxt-ripple/package.json:12]: both the folder and the export to be called `/lib` IMHO

Comment (jeffdowdle) [reply packages/nuxt-ripple/package.json]: Updated to use `/mapping/utils` as discussed

File: packages/nuxt-ripple/components/TideContentPage.vue [hunk 2]
```diff
@@ -41,12 +41,16 @@
...
 import { useTideSite, useTidePage } from '#imports'
-import { computed } from 'vue'
+import { computed, onMounted } from 'vue'
 import { pascalCase, pascalCaseTransformMerge } from 'change-case'
 
 const site = await useTideSite()
 const page = await useTidePage()
 
+onMounted(() => {
+  console.log(page)
```
Comment (lambry) [inline packages/nuxt-ripple/components/TideContentPage.vue:51]: @jeffdowdle A sneaky log

**Review by dylankelly — APPROVED** [review]

**Review by lambry — APPROVED** [review]

**Pull request merged**